src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -45,7 +57,26 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
   sys.addShutdownHook(shutdown())
 
   lazy val db = DatabaseUtil.db
-  val tezosNodeOperator = new TezosNodeOperator(tezosConf.network, batchingConf)
+
+  /* this is needed later to connect api logic to the fetcher,
+   * being dependent on the specific effect, i.e. Future, we can't
+   * devise a more generic solution, yet...
+   */
+  private[this] val fetchMaxLevel = ApiOperations.fetchMaxLevel _
+
+  //get all DataFetcheres and RemoteRpc instances needed in scope, for later use by the node methods
+  import TezosRemoteInstances.Akka.Futures._
```

Comment:
IntelliJ lists this as unused, how does Node find this instance in scope?

Reply:
I guess IDEA is wrong, you can try commenting out the line, and the ensuing error should point to the point of usage 😉